book/example_3_surface.md

```diff
@@ -19,6 +19,8 @@ channel, we can use the `tokio` library:
 ```rust, ignore
     let (input_example, example_recv) = tokio::sync::mpsc::unbounded_channel::<usize>();
 ```
+This is a multiple-producer/single-consumer (`mpsc`) channel, which is usually the appropriate choice for an inbound Hydroflow stream:
```

Comment:
Could link to https://docs.rs/tokio/latest/tokio/sync/mpsc/fn.unbounded_channel.html